app.json

```diff
@@ -1,8 +1,8 @@
 {
   "expo": {
-    "version": "2.0.3",
+    "version": "2.0.4",
     "ios": {
-      "buildNumber": "4"
+      "buildNumber": "1"
```

Comment:
I think Apple might complain because they always want a buildNumber higher than what was previously released 😮

Reply:
I love using auto increment for that https://docs.expo.dev/build-reference/app-versions/#remote-version-source

Reply:
But maybe we can add this later

Reply:
buildNumber needs to be higher based on the appVersion, so it should always increase the app version first, and then reset this to "1"

Reply:
yeah actually with that option set, you may be able to get rid of the extra line in the version bump script.